Userland/Applets/Audio/main.cpp

```diff
@@ -237,11 +240,7 @@ int main(int argc, char** argv)
     window->show();
 
     // This positioning code depends on the window actually existing.
-    if (!Config::read_bool("AudioApplet", "Applet", "ShowPercent", false)) {
-        window->resize(16, 16);
-    } else {
-        window->resize(44, 16);
-    }
+    dynamic_cast<AudioWidget*>(window->main_widget())->set_audio_widget_size(Config::read_bool("AudioApplet", "Applet", "ShowPercent", false));
```

Comment:
This should work with a `static_cast`. Also, please cast to a smart pointer or a reference.

Reply:
Since `window->main_widget()` returns a `Widget*` I think it is reasonable to cast it to `AudioWidget*` :)